src/client/dfs/obj.c

```diff
@@ -412,6 +412,12 @@ open_stat(dfs_t *dfs, dfs_obj_t *parent, const char *name, mode_t mode, int flag
 
 out:
 	if (rc == 0) {
+		if (flags & O_CREAT) {
+			DFS_OP_STAT_INCR(dfs, DOS_CREATE);
+		} else {
+			DFS_OP_STAT_INCR(dfs, DOS_OPEN);
+		}
```

Comment:
@mchaarawi: I will rework this to increment the counters in the relevant `open_` helpers. Not sure what I was thinking here, tbh...